src/prototype_rust_sdk/workflow_context.rs

```diff
@@ -162,11 +195,38 @@ impl WfContext {
 
     /// Request to run a local activity
     pub fn local_activity(
-        &mut self,
+        &self,
+        opts: LocalActivityOptions,
+    ) -> impl CancellableFuture<ActivityResolution> + '_ {
+        // We need to immediately create the first LA future to be consistent with the behavior
+        // of other functions where the command is scheduled before being awaited on.
+        let first_la_fut = self.local_activity_no_timer_retry(opts.clone());
+        ManualCancellableFuture::new(|_ct| async move {
```

Comment:
What is the purpose of your custom `ManualCancellableFuture` here if the cancellation token is unused?

Reply:
Still have to add that bit :)

Reply:
(Which I will do with next PR adding LA cancels since it'll just blow up right now trying to cancel them since it's unimplemented)